modules/appeals_api/spec/docs/v2/nod_spec.rb

```diff
@@ -151,27 +153,71 @@
     end
   end
 
-  path '/notice_of_disagreements/schema' do
-    get 'Gets the Notice of Disagreement JSON Schema.' do
-      tags 'Notice of Disagreements'
-      operationId 'nodSchema'
-      description 'Returns the [JSON Schema](https://json-schema.org/) for the `POST /notice_of_disagreement` endpoint.'
-      security [
-        { apikey: [] }
-      ]
-      produces 'application/json'
+  if DocHelpers.wip_doc_enabled?(:segmented_apis, true)
+    path '/schemas/{schema_type}' do
+      get 'Gets the Notice of Disagreement JSON Schema.' do
+        tags 'Notice of Disagreements'
+        description 'Returns the [JSON Schema](https://json-schema.org/) for the `POST /forms/10182` endpoint.'
```

Comment:
I like including the backticks around `POST /forms/10182` - I just noticed that they're missing in the equivalent description for HLR [here](https://github.com/department-of-veterans-affairs/vets-api/pull/10717/files#diff-39f0e6060ca7b1326e569973b31abb9e07537e03616ec2e8f16f2364e74ad288R350) though

Reply:
Good call! I'll add that to this PR as a fixup